### DISCUSSION TITLE
DataTable

### DISCUSSION THREAD

**Discussion by vopbs** [flet-dev/flet#1665] [Q&A]
### Question

I made a query code, but each time the content is cumulative, there will be multiple headings.
I want to do a clean up every time there are new results. I would like to ask how to achieve.

### ------------------------------------------------------

- [x] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by avilca** [comment]
You could use clear property or 

#clear controls
        control1.value = ""
        control2.value = ""
        control3.value = ""
         .
         .
         .
        etc
